apple-nav/src/components/NavItem.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+
+
+const NavItem=(props)=> {
+    return (
+      <div >
+       <img alt={props.item.navName}style={{width:"40px"}}src={props.item.img}/>
+      </div>
+    );
```

Comment:
You could also create separate CSS files or utilize styled components instead of placing styles inline directly into the `style` attribute.